ten-simple-rules-dockerfiles.Rmd

```diff
@@ -34,57 +35,57 @@ csl: plos.csl
 
 # Introduction {#introduction .unnumbered}
 
-Containerisation is a useful and powerful to handle increasingly complex virtual laboratories for computational sciences [@boettiger_introduction_2017].
-To some extent all sciences today use algorithms to analyse data [REF], so the majority of esearchers need skills to handle virtual laboratories to ensure understandability and reproducibility.
-This documentation and transparency concerns both authors and their own work, but also collaborators, reviewers, and new colleagues, to avoid needless duplication of work and brain-drain [REF].
-A core aspect of thorough documentation are well-defined computational environments and following best practices from software development [@taschuk_ten_2017].
-Containerisation can help with these needs.
-It provides portable computational environments that are built from human and machine readable instructions and are thus well documented and usable by many.
-These environments can then _host_ executable and human-readable notebooks with the scientific analyses [@rule_ten_2019] and make sharing of properly reproducible workflows (cf. Rule 10 of @sandve_ten_2013) much easier as the environments can closely resemble the original researchers virtual laboratory.
-
-While there are a few tutorials for using containers for reproducible research [^1], there is no extensive examination how to write the actual instructions to create the containers.
-Several platforms for facilitating reproducible research are built on top of containers [@brinckman_computing_2018; @code_ocean_2019; @simko_reana_2019; @jupyter_binder_2018; @nust_opening_2017], but usually and intentionally hide this complexity from the researcher.
-Not everybody needs to understand and write Dockerfiles, as is reflected in rule 1.
-However, as one can argue that _"the number of unique reserach environments approximates the number of researchers"_ [@nust_opening_2017], the researchers who do need to craft their own environment definition should follow practices taking into account the scientific context.
+With access to version control systems [REF GITHUB, GITLAB, etc.] it has become increasingly easy to not only share algorithms, but also instructions for building and testing.
+Within this content, it is suggested to include instructions for building containers to package software and data dependencies [@boettiger_introduction_2017].
+By way of providing these instructions along with thorough documentation, it's much more likely to be able to reproduce an analysis workflow.
+A container is a portabal, encapsulated environment that is built from human and machine readable instructions.
```

Comment:
typo: portabal -> portable

Reply:
Thanks! This was already merged so perhaps we can check for it in master?

Reply:
Here you go: https://github.com/nuest/ten-simple-rules-dockerfiles/pull/25

For future edits, please open PR, issue or similar (we shouldn't be commenting on closed and merged PR, as @nuest is likely to have already made changes).